avocado/core/task/runtime.py

```diff
@@ -98,12 +109,15 @@ def get_test_from_runnable(cls, runnable, no_digits, index, variant,
                     job_id=job_id)
         return cls(task)
 
+
+class PreRuntimeTask(RuntimeTask):
+
     @classmethod
-    def get_dependencies_form_runnable(cls, runnable, status_server_uri=None,
-                                       job_id=None):
-        """Creates runtime tasks for dependencies from runnable
+    def create_task_from_runnable(cls, pre_runnable, status_server_uri=None,  # pylint: disable=W0221
```

Comment:
Since the class name and since this is a class method, I would rename just to `from_runnable()` because of the usage:

`task = PreRuntimeTask.from_runnable()`